ui/src/containers/Rating/index.tsx

```diff
@@ -1,24 +1,114 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { observer } from 'mobx-react';
+import { useHistory, useParams } from 'react-router-dom';
 import { StarIcon } from '@patternfly/react-icons';
+import { IconSize } from '@patternfly/react-icons';
+import { Alert, AlertActionCloseButton, AlertGroup } from '@patternfly/react-core';
+import { useMst } from '../../store/root';
+import { assert } from '../../store/utils';
 import './Rating.css';
 
-const Rating: React.FC = () => {
+const Rating: React.FC = observer(() => {
+  const { user, resources } = useMst();
```

Comment:
Can we move this down on line no 41
and `const {resources} = useMst()` on line 71